test/pkg/environment/common/setup.go

```diff
@@ -59,16 +61,17 @@ var (
 		&schedulingv1.PriorityClass{},
 		&v1.Node{},
 		&v1beta1.NodeClaim{},
-		// TODO @njtran add KWOKNodeClass
+		&v1alpha1.KWOKNodeClass{},
 	}
 )
 
 // nolint:gocyclo
 func (env *Environment) BeforeEach() {
 	debug.BeforeEach(env.Context, env.Config, env.Client)
-
 	// Expect this cluster to be clean for test runs to execute successfully
 	env.ExpectCleanCluster()
+	env.TimeIntervalCollector.Reset()
```

Comment:
Should the reset be at the BeforeEach or in the AfterEach?

Reply:
We actually call it in the Record() call in the AfterEach, so we don't need this at all.